metpy/plots/skewt.py

```diff
@@ -786,6 +786,32 @@ def plot(self, u, v, **kwargs):
         u, v = delete_masked_points(u, v)
         return self.ax.plot(u, v, **line_args)
 
+    def wind_vectors(self, u, v, **kwargs):
+        r"""Plot u, v data as wind vectors.
+
+        Plots the wind data on the hodograph as acutal wind vectors.
```

Comment:
actual is misspelled, but I'm not sure it's the right word to use there.

"Plot the wind data as vectors for each level, originating at the origin."?